### DISCUSSION TITLE
章鱼星球（妖机版）成功安装 armbian

### DISCUSSION THREAD

**Discussion by yellow-shine** [ophub/amlogic-s9xxx-armbian#2255] [Amlogic]
分享自己成功armbian系统 安装到 章鱼星球 的 EMMC

# 环境信息
1. 章鱼星球（妖机版)  --  Octopus Planet(FAKE)
2. 使用镜像: Armbian_24.5.0_amlogic_s912_bullseye_5.15.155_server_2024.04.15

# 准备工作

1. 使用rufus烧录 img  到 u 盘
3. 插入刚刚烧录好的U盘（靠近网卡的那个USB口），插入键盘，插入hdmi线，插入网线
4. 按上电源线通电，此时章鱼星球从U盘启动，等待初始化
 

# 安装 Armbian 系统到 EMMC

1. 命令行输入 armbian-install
2. 选择安装序号202，Octopus Planet(FAKE)
3. armbian-config 配置网卡（可选）
4. poweroff 关闭电源，拔掉U盘
5. 重新插拔电源 重启 章鱼星球

**Comment by ophub** [comment]
感谢分享

**Comment by CnAiTewang** [comment]
铁子，Armbian_24.5.0_amlogic_s912_bullseye_5.15.155_server_2024.04.15，这个镜像还能分享一下，没找到！谢谢！

**Reply by PJ-568** [reply]
可尝试自行编译：https://github.com/ophub/amlogic-s9xxx-armbian/blob/main/documents/README.cn.md#5-%E7%BC%96%E8%AF%91%E7%B3%BB%E7%BB%9F
